src/sentry/workflow_engine/endpoints/project_detector_details.py

```diff
@@ -0,0 +1,71 @@
+from drf_spectacular.utils import extend_schema
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases import ProjectAlertRulePermission, ProjectEndpoint
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.api.serializers import serialize
+from sentry.apidocs.constants import (
+    RESPONSE_BAD_REQUEST,
+    RESPONSE_FORBIDDEN,
+    RESPONSE_NOT_FOUND,
+    RESPONSE_UNAUTHORIZED,
+)
+from sentry.apidocs.parameters import DetectorParams, GlobalParams
+from sentry.workflow_engine.endpoints.serializers import DetectorSerializer
+from sentry.workflow_engine.models import Detector
+
+
+@region_silo_endpoint
+@extend_schema(tags=["Workflows"])
+class ProjectDetectorDetailsEndpoint(ProjectEndpoint):
+    def convert_args(self, request: Request, detector_id, *args, **kwargs):
+        args, kwargs = super().convert_args(request, *args, **kwargs)
+        try:
+            kwargs["detector"] = Detector.objects.get(project=kwargs["project"], id=detector_id)
+        except Detector.DoesNotExist:
+            raise ResourceDoesNotExist
+
+        return args, kwargs
+
+    publish_status = {
+        "GET": ApiPublishStatus.EXPERIMENTAL,
+        "PUT": ApiPublishStatus.EXPERIMENTAL,
+        "DELETE": ApiPublishStatus.EXPERIMENTAL,
+    }
+    owner = ApiOwner.ALERTS_NOTIFICATIONS
+
+    # TODO: We probably need a specific permission for detectors. Possibly specific detectors have different perms
```

Comment:
Yeah, we'll want to basically replicate these permissions. I've just been using these for now since we know it's right.

Reply:
lol this is your comment from another endpoint copy/pasted